pkg/minishift/docker/docker.go

```diff
@@ -67,6 +67,10 @@ type DockerCommander interface {
 	// An error indicates that the remove failed.
 	Rm(container string) error
 
+	// GetID return the ID of the container for a given name which is present
+	// Any occurring error is also returned.
+	GetID(container string) (string, error)
```

Comment:
@praveenkumar  I am not sure the use of GetID here. What is the purpose of GetID?

Reply:
It returns the ID of the running container. since now origin container not controlling the openshift API but a named container (which not constant) have this info in docker you can get id using the wildcard for named container.